modules/Ecommerce/Order/Handlers/UpdateOrderHandler.php

```
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\Ecommerce\Order\Handlers;
+
+use Modules\Ecommerce\Order\Commands\UpdateOrderCommand;
+use Modules\Ecommerce\Order\Repositories\OrderRepository;
+
+class UpdateOrderHandler
+{
+    public function __construct(
+        private OrderRepository $repository,
+    ) {
+    }
+
+    public function handle(UpdateOrderCommand $updateOrderCommand)
+    {
+        $this->repository->updateOrder($updateOrderCommand->getId(), $updateOrderCommand->toArray());
```

Consider adding error handling around the repository call. What happens if the update fails? The method should either throw an exception or return a boolean to indicate success/failure.